content/10.diagnostics.md

```diff
@@ -2,81 +2,84 @@
+Molecular tests are often carried out from nasopharyngeal cavity samples, including throat wash and saliva [@doi:10/ggp4qx].
 
-##### RT-PCR 
+##### RT-PCR
 
-The Drosten Lab, from Germany, was the first lab to establish and validate a diagnostic test to detect SARS-Cov-2. 
-This test uses real-time PCR (RT-PCR) [@doi:10.2807/1560-7917.ES.2020.25.3.2000045] to detect several regions of the viral genome: the orf1b of the RNA dependent RNA polymerase (RdRP), the Envelope protein gene (E), and the Nucleocapsid protein gene (N). 
-In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents. 
-Specifically this tests utilizes two probes against RdRP of which one is specific to SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2020.25.3.2000045]. 
+The Drosten Lab, from Germany, was the first lab to establish and validate a diagnostic test to detect SARS-CoV-2.
+This test uses real-time reverse transcription PCR (RT-PCR) [@doi:10.2807/1560-7917.ES.2020.25.3.2000045] to detect several regions of the viral genome: the orf1b of the RNA dependent RNA polymerase (RdRP), the Envelope protein gene (E), and the Nucleocapsid protein gene (N).
+In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
+Specifically this tests utilizes two probes against RdRP of which one is specific to SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
 Importantly, this assay did not give any false positive results.
 
-##### qRT-tPCR
+##### qRT-PCR
 
-Around the same time, researchers in China developed a Quantitative Real Time Reverse Transcription PCR (qRT-tPCR) test to identify two region of the viral genome, orf1b and N region [@doi:10.1093/clinchem/hvaa029]. 
-Specifically, this assay was tested on samples coming from two COVID-19 patients, including a panel of positive and negative controls consisting of RNA extracted from several cultured viruses. 
-The assay uses the N gene to screen patients, while the orf1b is used to confirm the infection [@doi:10.1093/clinchem/hvaa029].
-In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2 is one. 
-Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2. 
-However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade. 
+Around the same time, researchers in China developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two region of the viral genome, orf1b and N region [@doi:10.1093/clinchem/hvaa029].
```

Comment:
https://www.ncbi.nlm.nih.gov/probe/docs/techqpcr/ lists "real-time quantitative reverse transcription".  I'm not sure what the usual convention is.

Reply:
Indeed, both descriptions seem to be in use. The same goes for the acronym, qRT-PCR and RT-qPCR. Not sure what the preferred version is.

Reply:
Because I'm not an authority, let's go with the version you used.